packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AbstractModal from 'terra-abstract-modal';
+import Button from 'terra-button';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import { elementType } from 'airbnb-prop-types';
+import styles from './NotificationDialog.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Title of the notification-dialog.
+   */
+  title: PropTypes.string,
+  /**
+   * Message of the notification-dialog.
+   */
+  message: PropTypes.string,
+  /**
+   * Array of buttons. Restricted to terra-button.
+   */
+  actions: PropTypes.arrayOf(elementType(Button)),
```

Comment:
Curious, what's the reason for pulling in airbnb-prop-types to use`elementType`, versus using prop-type's `instanceOf`?

Reply:
This is changed completely.